packages/std/src/testing/mock.rs

```diff
@@ -893,6 +908,40 @@ impl StakingQuerier {
     }
 }
 
+#[cfg(feature = "cosmwasm_1_3")]
+#[derive(Clone, Default)]
+pub struct DistributionQuerier {
+    withdraw_addresses: HashMap<String, String>,
+}
+
+#[cfg(feature = "cosmwasm_1_3")]
+impl DistributionQuerier {
+    pub fn new(withdraw_addresses: HashMap<String, String>) -> Self {
+        DistributionQuerier { withdraw_addresses }
+    }
+
+    pub fn set_withdraw_addresses(&mut self, withdraw_addresses: HashMap<String, String>) {
+        self.withdraw_addresses = withdraw_addresses;
+    }
```

Comment:
Let's create a `set_withdraw_address` too. Then I suggest changing this interface to something like

```
pub fn add_messages<M: Into<CosmosMsg<T>>>(self, msgs: impl IntoIterator<Item = M>) -> Self {
```

and then call `set_withdraw_address` for every entry.

We could have a `DistributionQuerier::clear` for clearing.